pkg/haproxy/template/funcmap.go

```diff
@@ -57,6 +58,11 @@ func createFuncMap() gotemplate.FuncMap {
 			}
 			return out
 		},
+		"haquote": func(s string) string {
+			// put in single quotes
+			// escape single quotes inside as ' "'" ' (without the spaces)
+			return "'" + strings.ReplaceAll(s, "'", "'\"'\"'") + "'"
```

Comment:
Maybe `'"'"'` with backquote around makes it less unclear?

Reply:
good idea, should be good now